downstream/assemblies/platform/assembly-controller-projects.adoc

```diff
@@ -1,14 +1,98 @@
 [id="controller-projects"]
-
+ifdef::controller-GS[]
 = Managing projects
+endif::controller-GS[]
+ifdef::controller-UG[]
+= Projects
+endif::controller-UG[]
 
-A Project is a logical collection of Ansible playbooks, represented in the controller.
+A Project is a logical collection of Ansible playbooks, represented in {ControllerName}.
 You can manage playbooks and playbook directories different ways:
 
 * By placing them manually under the Project Base Path on your {ControllerName} server.
 * By placing your playbooks into a source code management (SCM) system supported by the {ControllerName}.
+ifdef::controller-GS[]
 These include Git, Subversion, and Mercurial.
+endif::controller-GS[]
+ifdef::controller-UG[]
+These include Git, Subversion, Mercurial and Red Hat Insights.
+
+For more information on creating a Red Hat Insights project, see [Setting up insights remediations].
+
+[NOTE]
+====
+The Project Base Path is `/var/lib/awx/projects`. However, this can be modified by the system administrator. 
+It is configured in `/etc/tower/conf.d/custom.py`. 
+
+Use caution when editing this file, as incorrect settings can disable your installation.
+====
+
+The Projects page displays the list of the projects that are currently available. 
+
+{ControllerNameStart} provides you with a *Demo Project* that you can work with initially.
+
+image::controller-demo-project-details.png[Demo Project]
+
+image:projects-list-all.png[Projects - home]
+
+The default view is collapsed (*Compact*) with project name and its status, but you can use the *>* next to each entry to expand for more information.
+
+image:projects-list-all-expanded.png[Projects - expanded]
+
+For each project listed, you can get the latest SCM revision image:sync.png[Refresh,15,15], edit image:leftpencil.png[Edit,15,15] the project, or copy image:copy.png[Copy,15,15] the project attributes, using the icons next to each project. 
+
+Projects can be updated while a related job is running. 
+
+In cases where you have a large project (around 10 GB), disk space on `/tmp` may be an issue.
+
+*Status* indicates the state of the project and may be one of the following (note that you can also filter your view by specific status
+types):
+
+* *Pending* - The source control update has been created, but not queued or started yet. 
+Any job (not just source control updates) stays in pending until it is ready to be run by the system. 
+Possible reasons for it not being ready are:
+
+* Because it has dependencies that are currently running so it has to wait until they are done or there is 
```

Comment:
Suggested re-write:
* It has dependencies that are currently running so it has to wait until they are done.
* There is enough capacity to run in the locations it is configured to.